proto/worker.proto

```diff
@@ -202,3 +204,43 @@ message DebugStateReply {
 message PurgeTasksRequest {
     BigInt dealID = 1;
 }
+
+message WorkerMetricsRequest {}
+
+message GPUMetrics {
+    // pci bus id
+    string id = 1;
+    // human-friendly device name
+    string name = 2;
+    // core temperature, celsius
+    float temperature = 3;
+    // fan speed in percent from max
+    float fan = 4;
+    // power consumption, watts
+    float power = 5;
+}
+
+message CPUMetrics {
+    float utilization = 1;
+}
+
+message RAMMetrics {
+    uint64 total = 1;
+    uint64 free = 2;
+}
+
+message DiskMetrics {
+    uint64 total = 1;
+    uint64 free = 2;
+}
+
+message WorkerMetricsResponse{
```

Comment:
do we really need typed response here? Maybe somethig like {name, value, aggregation type}?

Reply:
This depends on how we will use them. If they are just for printing - then `map<string, NamedMetric>` is OK. Otherwise, it is better to have typed metrics like here.